kuma/wiki/migrations/0003_auto_20150703_0419.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
```

Comment:
This migration file is basically changes that we've done in the past weeks and gone unnoticed, e.g. extension of the locale setting, change of the related names of the tag fields.